<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editdeckwindow</class>
 <widget class="QDialog" name="editdeckwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fTitleLabel">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="fTitle"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="fAuthorLabel">
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="fAuthor"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="fDescriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="fDescription"/>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="fButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
